### ISSUE TITLE
ERROR:  Unable Registry Docker

### ISSUE THREAD

**Issue by pyrodie18** [Security-Onion-Solutions/securityonion#3900]
**System**:  Centos 7 clean install and fully updated (VM)
**Error**:  During install receive the following error.... `[ERROR   ] Failed to pull ghcr.io/security-onion-solutions/registry:latest: Error 500: Get https://ghcr.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
     Comment: Failed to pull ghcr.io/security-onion-solutions/registry:latest: Error 500: Get https://ghcr.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
`
**Steps to Reproduce**:
1. Install Centos 7
2. yum update -y
3. git clone https://github.com/Security-Onion-Solutions/securityonion
4. bash so-setup-network
5. Select eval version
6. Select all defaults

**Comment by jertel** [comment]
Is this still happening? Most likely GitHub's Container Registry was suffering from internal errors that have since been resolved.

**Comment by pyrodie18** [comment]
Nope, no love.  My co-worker was able to install just fine on his test VM so not sure what is going on.  I've attached the sosetup.log file so maybe it will help you out.  The is a completely clean install of Centos 7 and the only thing I've done is a yum update before I followed the Sec Onion instructions (yum install git, git clone, ....)

[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/6339591/sosetup.log)
